src/DwhManager.php

```diff
@@ -384,4 +387,21 @@ private function generatePassword(): string
             ->generateString(self::GENERATED_PASSWORD_LENGTH);
         return $password;
     }
+
+    private function ensureCorrectDataRetention(): void
+    {
+        $query = sprintf(
+            'SHOW DATABASES LIKE %s',
+            $this->connection->quote($this->database)
+        );
+        $databaseInfo = $this->connection->fetchAll($query);
+        $retentionTime = $databaseInfo[0]['retention_time'];
+        if ($retentionTime > self::DATABASE_DEFAULT_DATA_RETENTION_DAYS) {
```

Comment:
když je to `ensureCorrectDataRetention`, tak bych čekal spíš

```php
if ($retentionTime != self::DATABASE_DEFAULT_DATA_RETENTION_DAYS) {
```

jinak by se ta medota měla spíš jmenovat `enusreMaximumDataRetention`?